plugins/talk-plugin-featured-comments/client/components/Comment.js

```diff
@@ -22,9 +22,14 @@ class Comment extends React.Component {
         </blockquote>
 
         <div className={cn(`${pluginName}-comment-username-box`)}>
-          <strong className={cn(styles.username, `${pluginName}-comment-username`)}>
-            {comment.user.username}
-          </strong>
+
+          <Slot
+            className={cn(styles.username, `${pluginName}-comment-username`)}
+            fill="commentAuthorName"
+            comment={comment}
```

Comment:
Slot Requirements:
- The slot component needs the queryData `{root, comment, asset}`
- The slot fragments needs to be added to the fragments tree.
- Current workaround also requires `data={data}` to be passed to the Slot.
- A default component also needs to be added